tests/ragger/app/client.py

```diff
@@ -23,15 +56,20 @@ class   EthereumClient:
         ),
         SettingType.VERBOSE_EIP712: SettingImpl(
             [ "nanox", "nanosp" ]
+        ),
+        SettingType.VERBOSE_ENS: SettingImpl(
+            [ "nanox", "nanosp" ]
         )
     }
     _click_delay = 1/4
     _eip712_filtering = False
 
-    def __init__(self, client: BackendInterface):
+    def __init__(self, client: BackendInterface, golden_run: bool):
         self._client = client
+        self._chain_id = 1
         self._cmd_builder = EthereumCmdBuilder()
         self._resp_parser = EthereumRespParser()
+        self._nav = NanoNavigator(client, client.firmware, golden_run)
```

Comment:
Not sure I like this, I think that navigation is test-specific so it should stay in the test code and not the app plugin